File: Auto-generated-tasks/Difficulty-3/Task-3/Solution.pas
Var
	Count := 5, h, R := 0: integer;
	A: Array[0..Count] of integer;
Begin
	for h := 0 to Count do begin
		A[h] := Random(812) + 8;
		WriteLn("A[", h, "]=", A[h]);
		R := R + A[h];
	end;
	R := R / Count;
	WriteLn("R=", R)
End.

File: Auto-generated-tasks/Difficulty-3/Task-3/Solution.py
from random import *
Count = 5
h = 0
R = 0
A = [0] * Count
for h in range(0, Count):
	A[h] = randint(8, 820)
	print("A[", h, "]=", A[h])
	R = R + A[h]
R = R / Count
print("R=", R)